// @civic/common-gateway-react/dist/esm/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.ButtonMode = exports.getTokenDescription = exports.IdentityButton = exports.GatewayProvider = exports.useGateway = exports.logger = exports.ChainType = exports.CivicPassIssuanceStatus = exports.TokenIssuanceState = exports.GatewayStatus = exports.GatekeeperAPIStatus = exports.TokenState = exports.State = void 0;
/* eslint-disable no-console */
/* eslint-disable global-require */
/* eslint-disable @typescript-eslint/no-var-requires */
var types_1 = require("./types");
Object.defineProperty(exports, "State", { enumerable: true, get: function () { return types_1.State; } });
Object.defineProperty(exports, "TokenState", { enumerable: true, get: function () { return types_1.TokenState; } });
Object.defineProperty(exports, "GatekeeperAPIStatus", { enumerable: true, get: function () { return types_1.GatekeeperAPIStatus; } });
Object.defineProperty(exports, "GatewayStatus", { enumerable: true, get: function () { return types_1.GatewayStatus; } });
Object.defineProperty(exports, "TokenIssuanceState", { enumerable: true, get: function () { return types_1.TokenIssuanceState; } });
Object.defineProperty(exports, "CivicPassIssuanceStatus", { enumerable: true, get: function () { return types_1.CivicPassIssuanceStatus; } });
Object.defineProperty(exports, "ChainType", { enumerable: true, get: function () { return types_1.ChainType; } });
var logger_1 = require("./logger");
Object.defineProperty(exports, "logger", { enumerable: true, get: function () { return __importDefault(logger_1).default; } });
var GatewayContext_1 = require("./gateway/GatewayContext");
Object.defineProperty(exports, "useGateway", { enumerable: true, get: function () { return GatewayContext_1.useGateway; } });
Object.defineProperty(exports, "GatewayProvider", { enumerable: true, get: function () { return GatewayContext_1.GatewayProvider; } });
var button_1 = require("./button");
Object.defineProperty(exports, "IdentityButton", { enumerable: true, get: function () { return button_1.IdentityButton; } });
Object.defineProperty(exports, "getTokenDescription", { enumerable: true, get: function () { return button_1.getTokenDescription; } });
Object.defineProperty(exports, "ButtonMode", { enumerable: true, get: function () { return button_1.ButtonMode; } });
